app/src/ui/add-repository/add-existing-repository.tsx

```diff
@@ -82,14 +85,33 @@ export class AddExistingRepository extends React.Component<
       return
     }
 
+    const isBare = await isBareRepository(this.state.path)
+    if (isBare !== null && isBare === true) {
```

Comment:
Returning just a `bool` cleans this up as `null` doesn't seem to serve any purpose here.